### PR TITLE
Added any key released condition

### LINKED ISSUE #1510: Add a condition for "Any key released"
Hello, also I did talk about me asking about this and if it was added and that and I did say I would request it.

## Description
It's a request to add a new condition and I want it to be added because of.

1. I know it's not there.

2. I really need it so I can make a check if I released a key (for example) 3 times.

## Solution suggested

I got given some code so I could add it to an extension and the code is. 
```JS
eventsFunctionContext.returnValue = false;
for(var keyCode in this._releasedKeys.items) { 
if (this._releasedKeys.items.hasOwnProperty(keyCode)) { 
   if (this._releasedKeys.items[keyCode]) {
       eventsFunctionContext.returnValue = true; 
   }
 }
} 
```

The person who gave me the code was arhturo555.

## The thing I want to be added. 

What I want to be added is Any key released.

Any key released will be like Any key pressed and the key is released.

## Alternatives considered
I only know of one way that it works and that is that code that arhturo555 gave me.

I hope this makes sense also I only tried again because I asked if I should and the person said yes but try doing it a bit better.

### PR DESCRIPTION
**Pull request by Oxey405**
Fixes #1510 
Hello !
I fixed the issue #1510 !
Now, there is a condition in the keyboard folder named "Any Key released".
I hope this will help ! Have a nice day !
( thank you to @arthuro555 for helping me understanding the GDevelop 5 architecture)

### CODE REVIEW

**Comment by Oxey405** [comment]
YESS !!! WORKING !!!!

File: GDJS/Runtime/inputmanager.ts [hunk 1]
```diff
@@ -126,7 +126,7 @@ namespace gdjs {
      * Return true if the key corresponding to the location-aware keyCode was released during the last frame.
      * @param locationAwareKeyCode The location-aware key code to be tested.
      */
-    wasKeyReleased(locationAwareKeyCode: number) {
+    wasKeyReleased(locationAwareKeyCode: string | number) {
```
Comment (4ian) [inline GDJS/Runtime/inputmanager.ts:129]: Is this change necessary? locationAwareKeyCode should always be numbers, never strings.

File: Core/GDCore/Extensions/Builtin/KeyboardExtension.cpp [hunk 2]
```diff
@@ -82,6 +82,16 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsKeyboardExtension(
                     "res/conditions/keyboard.png")
       .AddCodeOnlyParameter("currentScene", "");
 
+ extension
+      .AddCondition("AnyKeyReleased",
+                    _("Any key Released"),
```
Comment (4ian) [inline Core/GDCore/Extensions/Builtin/KeyboardExtension.cpp:87]: `Any key released`

(careful about capitalization)

**Review by 4ian — COMMENTED** [review]
Thanks for the PR:) Almost perfect, great job. Added a few comments :)

File: GDJS/Runtime/inputmanager.ts (hunk 1 above)
Comment (Oxey405) [reply GDJS/Runtime/inputmanager.ts:129]: No it's not. I added it when arthuro helped me working on the test of this new feature but it's now useless you can remove it if  you want :D

File: Core/GDCore/Extensions/Builtin/KeyboardExtension.cpp (hunk 2 above)
Comment (Oxey405) [reply Core/GDCore/Extensions/Builtin/KeyboardExtension.cpp:87]: Oh yes excuse me. I am french (like you) so I put capital at begining of a sentence.

Comment (Oxey405) [reply Core/GDCore/Extensions/Builtin/KeyboardExtension.cpp:87]: I am correcting it !

Comment (Oxey405) [reply Core/GDCore/Extensions/Builtin/KeyboardExtension.cpp]: Done

Comment (Oxey405) [reply Core/GDCore/Extensions/Builtin/KeyboardExtension.cpp]: Oh crap I erased the head by accident :/ I think it's not a big deal

Comment (4ian) [reply Core/GDCore/Extensions/Builtin/KeyboardExtension.cpp]: It's not a problem. Prefer batching your answer into a single one because otherwise I'm spammed with notifications. Thanks :)

File: GDJS/GDJS/Extensions/Builtin/KeyboardExtension.cpp [hunk 3]
```diff
@@ -22,6 +22,8 @@ KeyboardExtension::KeyboardExtension() {
       "gdjs.evtTools.input.wasKeyReleased");
   GetAllConditions()["AnyKeyPressed"].SetFunctionName(
       "gdjs.evtTools.input.anyKeyPressed");
+        GetAllConditions()["AnyKeyReleased"].SetFunctionName(
```
Comment (4ian) [inline GDJS/GDJS/Extensions/Builtin/KeyboardExtension.cpp:25]: This is not properly aligned, can you fix the extra spaces?

Comment (Oxey405) [reply GDJS/GDJS/Extensions/Builtin/KeyboardExtension.cpp:25]: Okay I'm gonna work on it

**Comment by 4ian** [comment]
Thanks 👍👍
